meta/main.yml

```diff
@@ -24,11 +24,11 @@ galaxy_info:
     - ubuntu
 
 dependencies:
-  - src: kamaln7.swapfile
-    version: "0.4"
-    swapfile_size: "{{ sb_debian_base_swap_file_size }}"
+  - src: geerlingguy.swap
+    swap_file_size_mb: "{{ sb_debian_base_swap_file_size }}"
     become: yes
-    when: bootstrap is defined
+    when: |
+      bootstrap is defined
```

Comment:
@speedlight is this line break necessary?

Reply:
No is not necessary, is my habit to put conditionals in separates lines :) I fixed this, thanks for the observation.